### PR TITLE
Add DELL ForceTen S4810 CPU / Memory support

### PR DESCRIPTION
**Pull request by NerdBlender**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Fixed CPU and mem for Sxxxx series switches, and CPU for Cxxxx Series 

Should be easy to add support for M, E and Z series switches, but I don't have snmp info for those models

### CODE REVIEW

File: includes/discovery/mempools/dnos.inc.php [hunk 1]
```diff
@@ -5,9 +5,19 @@
 if ($device['os'] == 'dnos') {
     echo 'DNOS-MEMORY-POOL:  ';
 
-    $free = snmp_get($device, '.1.3.6.1.4.1.674.10895.5000.2.6132.1.1.1.1.4.1.0', '-OvQ');
-
-    if (is_numeric($free)) {
-        discover_mempool($valid_mempool, $device, 0, 'dnos-mem', 'Memory Utilization', '1', null, null);
+    $get_series = explode('.', snmp_get($device, 'mib-2.1.2.0', '-Onvsbq', 'F10-PRODUCTS-MIB', 'dnos'), 2); // Get series From MIB
```
Comment (laf) [inline includes/discovery/mempools/dnos.inc.php]: Does it not have a better named OID instead of `mib-2.1.2.0`?

Comment (NerdBlender) [inline includes/discovery/mempools/dnos.inc.php]: Unfortunately not:
SNMP Walk gives this

SNMPv2-SMI::mib-2.1.2.0 = OID: F10-PRODUCTS-MIB::f10SSeriesProducts.14

I could change it to SNMPv2-SMI::mib-2.1.2.0 if that reads better?

Comment (laf) [inline includes/discovery/mempools/dnos.inc.php]: I'm wondering if it's better to just use the numerical OID instead.

File: includes/discovery/mempools/dnos.inc.php [hunk 2]
```diff
@@ -5,9 +5,19 @@
...
+    $series = $get_series[0];
+    if ($series == 'f10SSeriesProducts') {
+        $total = snmp_get($device, 'F10-S-SERIES-CHASSIS-MIB::chSysProcessorMemSize.1', '-OvQU');
```
Comment (laf) [inline includes/discovery/mempools/dnos.inc.php]: You can move the F10-S-SERIES-CHASSIS-MIB into the call so for this:

`$total = snmp_get($device, 'chSysProcessorMemSize.1', '-OvQU', 'F10-S-SERIES-CHASSIS-MIB');`

Please do this for all of them.

**Comment by laf** [comment]
Thanks for contributing :)

**Pull request merged**